fdbclient/SpecialKeySpace.actor.cpp

```diff
@@ -46,7 +46,7 @@ std::unordered_map<std::string, KeyRange> SpecialKeySpace::managementApiCommandT
 	                .withPrefix(moduleToBoundary[MODULE::MANAGEMENT].begin) }
 };
 
-std::unordered_set<std::string> SpecialKeySpace::options = { "exclude/force", "failed/force" };
```

Comment:
Original implementation has `\xff\xff/management/excluded/` and `\xff\xff/management/options/exclude/`, which is not consistent to use.
Make both `excluded` now.